### ISSUE TITLE
use local timezone 

### ISSUE THREAD

**Issue by govvin** [Zverik/osmstreak#8]
Allow user to specify timezone to define "today".

**Comment by Zverik** [comment]
I agree that this is inconvenient — although it only matters in the telegram bot reminders. OSM API uses UTC, hence "today" is always in UTC.

**Comment by b-jazz** [comment]
What matters is that I might make an addition one morning, and then the following evening, but that will be counted across three days instead of two and I will have broken my "streak" when I really have mapped every "day".
